openpype/pype_commands.py

```diff
@@ -384,6 +384,30 @@ def run_tests(self, folder, mark, pyargs,
         import pytest
         pytest.main(args)
 
+    @staticmethod
+    def set_value(set_url, project=None):
```

Comment:
I would change the function names to `get_secure_value` and `set_secure_value` as `get_value` won't tell much. Also these functions should be just wrappers for functions in lib that contain the logic so we can call it from OpenPype directly if needed like when MongoURL is changed from local settings.

Reply:
Sounds reasonable, changed.